File: packages/graphql-codegen-cli/tests/test-files/dart2.dart
String gql(String input) => input;

String readRepositories = gql("""
  query ReadRepositories(\$nRepositories: Int!) {
    viewer {
      repositories(last: \$nRepositories) {
        nodes {
          id
          name
          viewerHasStarred
        }
      }
    }
  }
""").replaceAll('\n', ' ');

File: packages/graphql-codegen-cli/tests/test-files/dart2.ts
import gql from 'graphql-tag';

export const myQuery = gql`
  query ReadRepositories($nRepositories: Int!) {
    viewer {
      repositories(last: $nRepositories) {
        nodes {
          id
          name
          viewerHasStarred
        }
      }
    }
  }
`;
